CMSgov/mint-app
# [EASI-4205] Avatar component

# EASI-4205


## Changes and Description

- Added new `Avatar` component to handle the avatar circles and rendering username
- Added more colors 
- Deprecated `IconInitial` and `AssessmentIcon` components
- Updated various compoents from using the deprecated components



## How to test this change

1. Check Notification Center, discussion center, task list locking, the collaborator on task list section,  and the collaborator table when you add new collaboator to verify the colors of the avatar are consistent
2. Try assessment user to verify the avatar is that special star assessment icon (![image](https://github.com/CMSgov/mint-app/assets/2349702/495ed925-631d-41d2-b7a4-289e2b2cffd0))
3. Verify the MINT system admin that sends the daily digest in notification center has the new "eco" icon (![image](https://github.com/CMSgov/mint-app/assets/2349702/ac1cfeef-6078-49fa-993a-5b0cab746b17))



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
